<launch>
    <node name = "segmented_camera_info_publisher" pkg = "igvc_platform" type = "segmented_camera_info_publisher" output = "screen">
        <rosparam param="camera_names">["/cam/center", "/cam/left", "/cam/right"]</rosparam>
        <rosparam param="semantic_info_topic_prefix">["", "", ""]</rosparam>
        <rosparam param="semantic_info_topic_suffix">["/raw/info", "/raw/info", "/raw/info"]</rosparam>

        <param name="image_info_base_topic" value="/raw/info" />
        <param name="segmented_publisher_path" value="/segmented/info" />

        <param name="output_width" value="600" />
        <param name="output_height" value="480" />
    </node>
</launch>